image/pro_base/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM seafileltd/base:16.04
+
+# syslog-ng and syslog-forwarder would mess up the container stdout, not good
+# when debugging/upgrading.
+RUN apt update
+
+RUN apt-get install -y openjdk-8-jre libmemcached-dev zlib1g-dev pwgen curl openssl poppler-utils libpython2.7 libreoffice \
```

Comment:
what about the locale? At least on debian I always instale locales and set the locale to en_US utf8 to prevent locale issues (e.g. wrong handling of utf8 characters).